source/install/requirements.rst

```diff
@@ -44,7 +44,7 @@ PC Web
     :header: "Browser", "Technical Requirement"
 
     "Chrome", "v77+"
-    "Firefox", "v60+"
+    "Firefox", "v68.3", "v71.0+"
```

Comment:
May be change to

„Latest ESR or newer“

? Then you don’t need to change this every 6 weeks

Reply:
Agree on the policy of the latest ESR and later (I think this was the intent, but I wrote incorrectly in the process doc..my mistake).

With that, we could say "v68+", which would stay as the supported version until September/2020 when the next ESR is released.

![image](https://user-images.githubusercontent.com/13119842/71042430-b98a1000-20f9-11ea-9b90-dba0f4842744.png)

@amyblais I would also recommend checking the data each time. It can be a short one each time, but it's helpful to make sure we don't have a lot of large customers using Firefox 60. If yes, we may need to reach out to them to let them know about the version requirement change.